cmd/tls/install/install.go

```diff
@@ -32,6 +35,7 @@ const (
 // NewTLSInstallCmd returns the tls install command.
 func NewTLSInstallCmd() *cobra.Command {
 	options := tls.Options{}
+	options.Printer = output.NewPrinter("tls", true, false, os.Stdout)
```

Comment:
I suggest not initializing the printer inside the `tls` command, but passing it through the `common.Options` as we do for the other subcommands: https://github.com/falcosecurity/falcoctl/blob/7227ec4afdafeec4d316e13e0bdc95c904604923/cmd/root.go#L58-L61

This way we ensure that the printer is configured using the global flags such as `verbosity & styling`. In order to configure the printer you can still use the https://github.com/falcosecurity/falcoctl/blob/7227ec4afdafeec4d316e13e0bdc95c904604923/pkg/options/common.go#L80 function.

Reply:
@alacuku i see, thank you for the feedback! will make the changes